samples/bluetooth/mesh/light_ctrl/src/main.c

```diff
@@ -14,6 +14,46 @@
 #include "model_handler.h"
 #include "lc_pwm_led.h"
 
+#ifdef CONFIG_EMDS
+#include <emds/emds.h>
+#endif
+
+#ifdef CONFIG_EMDS
+#define EMDS_DEV_IRQ 24 /* device uses IRQ 24 */
+#define EMDS_DEV_PRIO 0 /* device uses interrupt priority 2 */
+#define EMDS_ISR_ARG 0
+#define EMDS_IRQ_FLAGS 0 /* IRQ flags */
```

Comment:
comments should be informative. No reason to have comments those reproduce definitions' names.